### DISCUSSION TITLE
Releases Documentation ambiguity

### DISCUSSION THREAD

**Discussion by DistroAgnostic** [tribalfs/GalaxyMaxHzPub#226] [General]
When I saw releases tab & found out that it was last updated 2 years ago then I was about to close the repo as I thought i already have Installed this app in my phone but then I thought app update notification came then I goto releases then again it still shows 27Jan last update date then I thought it was a bug was almost about the closes releases tab like I previously did then I scroll down & found that source code was of 2 years ago but then I saw apk was uploaded few hrs ago and then I downloaded. It's a bad practice to release apk with 2years old source code & new apk. Just give apk in a new release version so that no one will think that this repo is dead like hundreds of thousands of other repos on git[hub/lab/etc].

**Comment by tribalfs** [comment]
Thank you. I understand but this repo is not open sourced and is meant to host apks and app info only. New updates are designed to be notified in-app. I have to keep the same download link to simplify some stuff.

**Reply by DistroAgnostic** [reply]
Then Add release data in Release info of release section.
